docs/en/enterprise-edition/policy-reference/azure-policies/azure-networking-policies/bc-azure-2-49.adoc

```diff
@@ -0,0 +1,68 @@
+
+== Azure Machine Learning workspace is not configured with overly permissive network access
```

Comment:
@tsmithv11  - seems a mistake: Azure Machine Learning workspace **is** configured with overly permissive network access

Reply:
You're right and OSS people caught this too. We removed the policy and will readd it later.